### PR TITLE
Board of directors meeting 2022-02-28

### PR DESCRIPTION
**Pull request by bvhme**
Meeting notes

-----
[View rendered organization/board-of-directors-meetings/2022-02-28-minutes.md](https://github.com/publiccodenet/about/blob/2022-02-28-board-of-directors/organization/board-of-directors-meetings/2022-02-28-minutes.md)
[View rendered organization/board-of-directors-meetings/index.md](https://github.com/publiccodenet/about/blob/2022-02-28-board-of-directors/organization/board-of-directors-meetings/index.md)

### CODE REVIEW

File: organization/board-of-directors-meetings/2022-02-28-minutes.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      1. Completed
+   5. Action: Course request, Boris to review the request with the submitter based on planning
+      1. Completed: coursework approved in line with what we discussed
+   6. Action: Boris to engage law firm based on his email
+      1. Completed
+   7. In April: Discuss who to hire next
+   8. When the lockdown in the Netherlands ends: Boris to resume recruiting the ‘medewerker administratie’
+2. For discussion:
+   1. Incoming documentation requests
+      1. Decision: we can re-engage external council
```
Comment (Ainali) [inline organization/board-of-directors-meetings/2022-02-28-minutes.md:24]: Is this referring to the strategic council or some other body? It looks very cryptic right now. If it needs to be secret, perhaps make that explicit like "re-engage [organization name redacted]"?

Comment (bvhme) [reply organization/board-of-directors-meetings/2022-02-28-minutes.md:24]: Great point, what about:

```suggestion
      1. Decision: we can re-engage external legal council
```

Comment (Ainali) [reply organization/board-of-directors-meetings/2022-02-28-minutes.md:24]: Yes, then it makes more sense!

File: organization/board-of-directors-meetings/2022-02-28-minutes.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+type: Resource
+---
+
+# Board meeting February 28th 2022
+
+1. Review action items:
+   1. Action: Eric to calculate the effect of the suggested salary policy on the burn rate
+      1. Unchanged: Blocked on needing a meeting with the bookkeeper
+   2. Action: Ben will revise the statement of support or partnership agreement based on the one sent to Montreal for approval by the board
+      1. Follow-up: Re-approach partner onboarding process.
```
Comment (Ainali) [inline organization/board-of-directors-meetings/2022-02-28-minutes.md:11]:
```suggestion
      1. Follow-up: Re-approach partner onboarding process
```

Comment (Ainali) [reply organization/board-of-directors-meetings/2022-02-28-minutes.md:11]: The only line with a period

**Review by ElenaFdR — DISMISSED** [review]
Ready to go, once Jan's change is made!

**Review by Ainali — APPROVED** [review]

**Pull request merged**